### PR TITLE
quincy: doc/radosgw: update S3 action list

### PR DESCRIPTION
**Pull request by zdover23**
Update the list of S3 actions supported in the Quincy release of Ceph.

Note: the list of S3 actions supported in Reef is not the list of S3 actions supported in Squid or in Reef.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/radosgw/bucketpolicy.rst [hunk 1]
```diff
@@ -36,58 +36,82 @@ For example, one may use s3cmd to set or delete a policy thus::
...
-- s3:PutBucketWebsite
-- s3:PutLifecycleConfiguration
-- s3:PutObjectAcl
-- s3:PutObject
-- s3:PutObjectVersionAcl
-- s3:PutReplicationConfiguration
-- s3:RestoreObject
+In Quincy, only the following actions are supported:
+
+.. note:: This list of S3 actions is accurate only for the Squid release of
```
Comment (anthonyeleven) [inline doc/radosgw/bucketpolicy.rst:41]: Do we want to say this in Quincy docs?

Comment (zdover23) [reply doc/radosgw/bucketpolicy.rst:41]: Aw man. Good catch. Serves me right for doing three things at the same time.

**Comment by anthonyeleven** [comment]
Gotta stock to only two :rolleyes:

Is the list itself Q-specific?

**Comment by zdover23** [comment]
> Gotta stock to only two :rolleyes:
> 
> 
> 
> Is the list itself Q-specific?

It is.

**Review by anthonyeleven — APPROVED** [review]

**Pull request merged**